src/lexer.mll

```diff
@@ -197,6 +199,8 @@ rule token = parse
       let posL = Range.from_lexbuf lexbuf in
       let strbuf = Buffer.create 128 in
       let (rng, s) = binary_literal posL strbuf lexbuf in
+      (* TODO: handle multi-codepoint unicode characters e.g. 🤷🏽‍♀️
+         (raise_error NotASingleCodePoint or convert somehow) *)
```

Comment:
This comment was not supposed to be here, this is about Strings, not Binaries. I will move it down to proper place.